microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,239 @@
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinnable music player app to play your favorite mp3 files",
+  "display": "browser",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+
+<!-- ====================================================================== -->
+## Detect usage in the sidebar
+
+Detecting when your app runs in the sidebar can be useful to provide your users with the best possible experience when your app runs in this specific browser UI surface.
+
+Starting with Microsoft Edge 114, you can detect when your app runs in the sidebar by using User-Agent Client Hints. To learn more about User-Agent Client Hints, see [Detecting Microsoft Edge from your website](../../web-platform/user-agent-guidance.md).
+
+To detect when your app runs in the sidebar, do one of the following:
+
+* On your web server, read the `Sec-CH-UA` HTTPS request header and look for the `Edge Side Panel` brand. For example:
+    
+    ```https
+    Sec-CH-UA: "Microsoft Edge";v="112", "Edge Side Panel";v="1", "Placeholder;Browser Brand";v="99"
+    ```
+
+* In the browser, use the `navigator.userAgentData` JavaScript API and read the value of the `brands` property. For example:
+
+    ```javascript
+    const brands = navigator.userAgentData.brands;
+    const sidebarBrandInfo = brands.find(b => b.brand === "Edge Side Panel");
+    if (sidebarBrandInfo) {
+      console.log(sidebarBrandInfo); // { brand: "Edge Side Panel", version: "1" }
+    } else {
+      console.log("App is not running in the Microsoft Edge sidebar");
+    }
+    ```
+
+You can also use the User Agent string to know whether your app runs in the sidebar or not.
+
+**Note**: we strongly recommend using User Agent Client Hints instead of the User Agent string. The User Agent string is an outdated mechanism for doing browser detection, and has website compatibility issues.
+
+If you can't use User Agent Client Hints in your app, look for `Edge Side Panel` in the User Agent string. Here are some examples of User Agent strings containing the `Edge Side Panel` value:
+
+```https
+User-Agent: ... (Edge Side Panel)...
+User-Agent: ... (..., Edge Side Panel)...
+User-Agent: ... (Edge Side Panel, ...)...
+```
+
+#### Relationship with your mobile app
+
+You might have created two variants of your app:
+
+* One variant when the app is used on a desktop device.
+* Another variant when the app is used on a mobile device.
+
+In this scenario, the desktop variant is used when your app is pinned to the sidebar. By default, apps in the sidebar that use the `edge_side_panel` manifest member receive the Microsoft Edge desktop User Agent Client Hint:
```

Comment:
```suggestion
Suppose you create a desktop variant of your app and a mobile variant of your app. In this scenario, the desktop variant is used when your app is pinned to the sidebar. By default, apps in the sidebar that use the `edge_side_panel` manifest member receive the Microsoft Edge desktop User Agent Client Hint:
```
"You might have created ...." is awkward and it sounds like speculating about user (avoid).
Simplified wording to make the concept seem simple and familiar, not requiring more precision.
removed blank line